### ISSUE TITLE
Gem installation fails because transitive dependencies don't support Ruby 1.9

### ISSUE THREAD

**Issue by magnusbaeck** [logstash-plugins/logstash-input-websocket#12]
The 2.5.0 release of the addressable library made on 2016-11-04 started requiring Ruby 2.0 compatibility (via the introduction of a dependency to the public_suffix library), so an attempt to do a bundle installation for this plugin started to result in the following:

```plain
$ bundle install
Fetching gem metadata from https://rubygems.org/.........
Fetching version metadata from https://rubygems.org/..
Fetching dependency metadata from https://rubygems.org/.
Resolving dependencies...........................................................................................................
public_suffix-2.0.4 requires ruby version >= 2.0, which is incompatible with the
current version, ruby 1.9.3p551 (jruby 1.7.26)
```

Since the ftw library that dragged in addressable hadn't locked down its dependency it, too, effectively started requiring Ruby 2.0. An attempt to fix this was made in https://github.com/jordansissel/ruby-ftw/pull/34, but it was insufficient and a follow-up PR (https://github.com/jordansissel/ruby-ftw/pull/35) is needed before we can step up the ftw dependency in this plugin to at least 0.0.46.

**Comment by jordansissel** [comment]
ruby ftw 0.0.46 published which should fix this.
